packages/material-ui/src/SnackbarContent/SnackbarContent.test.js

```diff
@@ -54,4 +54,11 @@ describe('<SnackbarContent />', () => {
       assert.strictEqual(wrapper.childAt(0).contains(message), true);
     });
   });
+
+  describe('prop: role', () => {
+    it('should render the role', () => {
+      const wrapper = shallow(<SnackbarContent message="message" role="alert" />);
```

Comment:
Since this is an a11y issue it's better to use `@testing-library/react` which treats ARIA attributes as the primary way to query for elements. I would rather see a `expect(queryByRole('alert')).to.have.text("message")` which is very close to how assistive technology would perceive this.

However, this test is still using `enzyme` which we're slowly phasing out. I'd be happy if you could follow up with a pull request that replaces all `enzyme` tests in this file with `@testing-library/react`. We use our own little wrapper that is located in `test/utils/createClientRender`. You can look at other tests for inspiration e.g. `Select.test.js`.

Reply:
Sure, I'd be happy to open another PR to convert the SnackbarContent test file. Should I leave this test as-is for right now and change it in the follow up PR, or change it in this PR?

Reply:
I'd prefer another PR. The test isn't blocking the general improvement of this change.